.github/PULL_REQUEST_TEMPLATE/design-document.md

```diff
@@ -0,0 +1,16 @@
+## Problem
```

Comment:
In current directory structure there are multiple templates possible. How the PR creator will be choosing the correct template?

Reply:
It's going to be setup in the repository configuration. If multiple are necessary, multiple can be configured. See how it works for issues now.